Gemfile

```diff
@@ -71,5 +71,6 @@ group :test do
   gem "webdrivers"
 end
 
```

Comment:
- [ ] It would be nice if you add a **Run tests** sub-section after your **Usage** sub-section to make your readme file more descriptive and professional :blush:

## Run tests
```
rspec spec
```